frontend/src/ui/ContentComponent.tsx

```diff
@@ -8,10 +8,10 @@ import format from "date-fns/format";
 import parseISO from "date-fns/parseISO";
 import * as React from "react";
 
-export interface ArticleContentProps
+export interface ContentComponentProps
   extends Pick<Article, "title" | "blocks" | "createdAt"> {}
 
-export const ArticleContentComponent: FC<ArticleContentProps> = ({
+export const ContentComponent: FC<ContentComponentProps> = ({
```

Comment:
może być na razie ale pewnie i tak trzeba będzie rozdzielić about i article bo do article coś dojdzie nowego, moim zdaniem jak już robić generycznie to lepiej byłoby wydzielić np sam wrapper  <div className="flex flex-1 flex-col p-4 lg:p-6 space-y-4 bg-gradient-to-b from-background-dark rounded-lg"> jako osobny komponent i użyć go w komponentachArticleContent i AboutContent